src/main/java/com/senlainc/warsaw/tyurin/util/propertyHandlers/PropertyReader.java

```diff
@@ -0,0 +1,38 @@
+package com.senlainc.warsaw.tyurin.util.propertyHandlers;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+
+    private static PropertyReader INSTANCE;
+    private Properties properties;
+
+    private PropertyReader() {
+        properties = new Properties();
+    }
+
+    public static PropertyReader getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new PropertyReader();
+        }
+        return INSTANCE;
+    }
+
+    public String readProperties(String propertyName, String path) {
```

Comment:
Also there is some performance issue - every time you try to get some property you rereading property file, i think it unnecessary, make it read property one time, and then get them from Property class